### ISSUE TITLE
Accessiblity: Headings not in logical order / misuse of (h1) heading tags

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#14]
WCAG 2.0 - Sucess criteria - 1.3.1 Info and relationships (Level A) violation
1. Go to:http://data.ottawa.ca/dataset
2. Under the 'Datasets' section in the search field type 'Parks and Rec"
3. Under the parks heading click the 'csv' icon
4. Parks download page will load for this file (http://data.ottawa.ca/dataset/parks/resource/3f15f808-51db-46b5-a1e2-a19a4dc48a09)
5. H1 should only be used once on the page. 

Resolution: Use H1 only once at the top of the page. 
![heading not logical order](https://cloud.githubusercontent.com/assets/5271253/8058976/4505ec02-0e8b-11e5-8b83-6b26c47134e9.JPG)

**Comment by deniszgonjanin** [comment]
@Dawnzar ready for review

**Comment by Dawnzar** [comment]
Retested: resolved.
